lib/lazy_doc.ex

```diff
@@ -1,7 +1,34 @@
 defmodule LazyDoc do
   @moduledoc """
   Documentation for `LazyDoc`.
+
+  This module is an example of autogenerated docs by
+  the task provided.
+  """
+
+  @doc """
```

Comment:
docs are written in markdown

what do you think about using a separate heading for Parameters and returns. for example
```

  ## Parameters

    - name: String that represents the name of the person.
    ```

Reply:
modifying the default prompt, i will do it in a separate PR.